Src/BootCamp.Chapter1/ArrayOperations.cs

```diff
@@ -51,8 +151,30 @@ public static int[] RemoveFirst(int[] array)
         /// <returns>A new array with element removed at a given index. If an array is empty or null, returns input array.</returns>
         public static int[] RemoveAt(int[] array, int index)
         {
-            // ToDo: implement.
-            return array;
+            if (index < 0 || index >= array?.Length)
+            {
+                return array;
+            }
+
+            if (ArrayIsNullOrEmpty(array))
+            {
+                return array;
+            }
+
+            int j = 0;
+            int[] auxArray = new int[array.Length - 1];
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['array' is null on at least one execution path.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4930349)